e2e/rbd.go

```diff
@@ -56,7 +56,7 @@ var (
 )
 
 func deployRBDPlugin() {
-	// delete objects deployed by rook
+	// delete objects deployed by rook.
```

Comment:
I think the comments-should-end-with-a-dot is only required for top-level comments, not inside functions? Could you check that please?

Reply:
Yes sure @nixpanic .  I will check and update accordingly.